frontend/public/locales/en/application.json

```diff
@@ -10,6 +10,19 @@
     "channel.type.label.chart": "Chart name",
     "channel.type.label.packageFilterVersion": "Package version",
     "channel.type.label.bucketPath": "Subfolder",
+    "delete.resource.modal.application.heading": "Permanently delete {0} application?",
```

Comment:
Given the translation effort that Robert is working on, I think we stops using application.json. Could you move the tests to where they are referred instead?

Reply:
Ah I see. Yep I think you could remove the application.json as no one is using it